lesson9-Logging/BrainstormSessions/Api/IdeasController.cs

```diff
@@ -99,15 +101,19 @@ public async Task<ActionResult<List<IdeaDTO>>> ForSessionActionResult(int sessio
         [ProducesResponseType(404)]
         public async Task<ActionResult<BrainstormSession>> CreateActionResult([FromBody]NewIdeaModel model)
         {
+            _logger.Debug($"Start of CreateActionResult Method execution with model {nameof(model)}");
+
             if (!ModelState.IsValid)
             {
+                _logger.Error($"ModelState {nameof(model)} is not valid");
```

Comment:
Same here. Also `ModelState` here represents a state of a model, with some properties and metadata of a model. So I'd not say that the model state is invalid, but model is invalid.

Reply:
done